manifests/compute/ec2.pp

```diff
@@ -5,9 +5,8 @@
 # === Parameters:
 #
 # $version::  Package version to install, defaults to installed
-#             type:Optional[String]
 #
-class foreman::compute::ec2($version = 'installed') {
+class foreman::compute::ec2(Optional[String] $version = 'installed') {
```

Comment:
I know it was already the case that it was optional, but since we have a default is it even possible to pass in undef somehow?